src/physics/navier_stokes.cpp

```diff
@@ -455,7 +442,7 @@ ::dissipative_source_term (
         for (int sr = 0; sr < nstate; ++sr) {
             real jac_grad_row = 0.0;
             for (int sc = 0; sc < nstate; ++sc) {
-                jac_grad_row += jacobian[sr][sc]*manufactured_solution_gradient[sc][d]; // CHANGED THE INDEXING HERE
```

Comment:
What was the motivation for swapping the index?

Reply:
Normally the columns correspond to the `x,y,z` directions and the rows correspond to the `state`, `euler.cpp` was modified to use this convention, and this also reduced repeated code in `navier_stokes.cpp`. See commit [`fda455b1621bcef40fd0333bd8b4ff06b0514f1f`](https://github.com/dougshidong/PHiLiP/pull/76/commits/fda455b1621bcef40fd0333bd8b4ff06b0514f1f)